tests/test_revisions.py

```diff
@@ -0,0 +1,124 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import requests
+import requests_mock
+
+LANDO_API_RESPONSE = {
```

Comment:
Can you put this in a new file like the `canned_responses` are structured in the lando api repo? This will help when we have more of these.

At the top of that new file you can add `yapf: disable` so that you can format the hash cleanly rather than have the linter mangle it as it has here.